src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState('');
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getPeople()
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);

  useEffect(() => {
    setIsLoading(true));
    
    getPeople()
```
You should set is loading state before loading started